### PR TITLE
debug_payment_bitcoin

### PR DESCRIPTION
**Pull request by dsolanki-initos**

### CODE REVIEW

**Comment by manengel** [comment]
@dsolanki-initos @jans23 Should this be reviewed, did this went forgotten? Or it is rather of low importance right now?

**Comment by jans23** [comment]
It is stalled for now and will be taken on again in the future. Nothing to do for you now.

File: payment_bitcoin/tests/test_payment_bitcoin.py [hunk 1]
```diff
@@ -0,0 +1,230 @@
+import codecs
+from datetime import datetime
+from datetime import timedelta as td
+from hashlib import sha256
+from ..models.bitcoin import validate_bitcoin_address, check_received
+
+DIGITS58 = "123456789ABCDEFGHJKLMNPQRSTUVWXYZabcdefghijkmnopqrstuvwxyz"
+CHARSET = "qpzry9x8gf2tvdw0s3jn54khce6mua7l"
```
Comment (jans23) [inline payment_bitcoin/tests/test_payment_bitcoin.py]: Looks confusing. Shouldn't this be alphabetically ordered?

Comment (dsolanki-initos) [reply payment_bitcoin/tests/test_payment_bitcoin.py]: I checked and it is redundant code I'll remove it.

**Pull request merged**